### PR TITLE
Docgen fixes

### PR DESCRIPTION
**Pull request by technoweenie**
A few minor tweaks. The output of `help subcmd` and `subcmd -h` are now identical. Also, leading and ending linebreaks are consistent.

/cc @sinbad

### CODE REVIEW

File: commands/commands.go [hunk 1]
```diff
@@ -224,7 +223,7 @@ func determineIncludeExcludePaths(includeArg, excludeArg string) (include, exclu
 
 func printHelp(commandName string) {
 	if txt, ok := ManPages[commandName]; ok {
-		fmt.Fprintf(os.Stderr, txt)
+		fmt.Fprintf(os.Stderr, "%s\n", strings.TrimSpace(txt))
```
Comment (sinbad) [inline commands/commands.go]: Hmm, this puts the command snug under the prompt which I find harder to read, e.g.:
```
steve@Stevens-MacBook-Pro ~/p/G/s/g/g/git-lfs> git lfs pull --help
git lfs pull [options] [<remote>]

Download Git LFS objects for the currently checked out ref, and update
```
I find this harder to read than
```
steve@Stevens-MacBook-Pro ~/p/G/s/g/g/git-lfs> git lfs pull --help

git lfs pull [options] [<remote>]

Download Git LFS objects for the currently checked out ref, and update
```

**Comment by sinbad** [comment]
:+1: I originally didn't remove the "Short" entries because Cobra generated a nice command list from them, but that's defunct since I started handling the root help topic. Nice catch on the br's.

File: commands/commands.go (hunk 1 above)
Comment (technoweenie) [inline commands/commands.go]: I agree that whitespace is usually good. However in this case, the empty space looks off to me:

```bash
$ git help
usage: git [--version] [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [-c name=value] [--help]
           <command> [<args>]

$ ruby -h
Usage: ruby [switches] [--] [programfile] [arguments]
  -0[octal]       specify record separator (\0, if no argument)
  -a              autosplit mode with -n or -p (splits $_ into $F)
```

**Comment by technoweenie** [comment]
Ok, merging this. We can always change the whitespace thing later :)

**Comment by sinbad** [comment]
Yeah, having reviewed other command help I see not having the spare line is normal, just looked odd to me after getting used to it being there :)

**Pull request merged**